Can't find system glfw3 on Ubuntu 14.10

Using `default-features = false` I'm getting `cannot find -lglfw3` when linking. The `libglfw3-dev` package installs `/usr/lib/x86_64-linux-gnu/libglfw.so`. The package conflicts with `libglfw-dev` (so you can't install both glfw 2 and 3 on ubuntu).

Is there a way to use libglfw.so as a fallback?

--- a/src/ffi/link.rs
+++ b/src/ffi/link.rs
@@ -19,6 +19,12 @@ extern {}
 
 #[cfg(not(feature = "glfw-sys"))]
 // leaving off `kind = static` allows for the specification of a dynamic library if desired
+#[cfg(target_family = "unix")]
+#[link(name = "glfw")]
+extern {}
+
+#[cfg(not(feature = "glfw-sys"))]
+#[cfg(target_family = "windows")]
 #[link(name = "glfw3")]
 extern {}
 

